test/e2e/curated_packages_test.go

```diff
@@ -40,12 +38,7 @@ func packageBundleURI(version v1alpha1.KubernetesVersion) string {
 }
 
 func runCuratedPackageInstallSimpleFlow(test *framework.ClusterE2ETest) {
-	os.Setenv(features.CuratedPackagesEnvVar, "false")
 	test.WithCluster(func(e *framework.ClusterE2ETest) {
-		os.Setenv(features.CuratedPackagesEnvVar, "true")
-		defer os.Setenv(features.CuratedPackagesEnvVar, "false")
-
-		e.InstallCuratedPackagesController()
```

Comment:
Is it ok removing this `e.InstallCuratedPackagesController()`  command, it looks like a real function?

Reply:
Yes checked with Eric on it.